packages/nextjs-mf/src/default-delegate.ts

```diff
@@ -6,15 +9,29 @@ import {
   WebpackRemoteContainer,
 } from '@module-federation/utilities/src/types/index';
 
+/**
+ * Initializing pure as RemoteVars
+ */
 let pure = {} as RemoteVars;
 try {
```

Comment:
The function 'extractUrlAndGlobal' does not handle the case where the '@' character is not found in the input string. This could lead to unexpected behavior.